src/modules/generalExtensions/assets/extensionCloseCall.ts

```diff
@@ -0,0 +1,23 @@
+export default (): void => {
+    const observer = new MutationObserver(handleObserved);
+    const observertarget = document.getElementById('radio_messages_important');
+    const observerOptions = {
+        childList: true,
+    };
+
+    function handleObserved() {
+        const msgClose = document.getElementsByClassName('radio_message_close');
+        for (let i = 0; i < msgClose.length; i++) {
+            let vid = msgClose[i].getAttribute('vehicle_id');
+            let prev = msgClose[i].previousElementSibling;
+            if (prev === null || vid === null) {
+                return;
+            }
+            prev.classList.add('radio_message_close');
+            prev.setAttribute('vehicle_id', vid);
+        }
+    }
+    if (window.location.pathname == '/' && observertarget != null) {
+        observer.observe(observertarget, observerOptions);
+    }
+};
```

Comment:
Having an click eventListener on `#radio_messages_important` would be easier. You could check if the target was a "to this mission" button :)

Also saves performance as we do not need a mutationObserver then :)

Reply:
should be changed with 
https://github.com/LSS-Manager/LSSM-V.4/pull/432/commits/0f195102ecccf35198bcc1b8a2560b073f2d87a4